src/src/WebServer/WebServer.cpp

```diff
@@ -408,6 +408,10 @@ void getWebPageTemplateDefaultHead(WebTemplateParser& parser, bool addMeta, bool
   if (parser.isTail()) return;
   parser.process(F("<!DOCTYPE html><html lang='en'>"
             "<head>"
+            "<link rel='stylesheet' href='codemirror.css'>"
```

Comment:
i guess this all must be in "WebStaticData" but this is beyond my capabilities... 😳

Reply:
It should have a `/` before that last `>` to make that `link` tag valid.

And the `script` tags below can be simplified by replacing `></script>` by `/>`

Reply:
Thanks... T

> It should have a `/` before that last `>` to make that `link` tag valid.

This works, there is no / missing (https://www.w3schools.com/css/css_howto.asp)

Reply:
Ah, then that's one of the peculiarities (read: diversions) from the regular XML-style rules that apply to the HTML dialect 💇‍♂️

Reply:
> And the script tags below can be simplified by replacing `></script>` by `/>`

You can scrape off a few bytes of the .bin size by applying this change (3x).

Would it work if you added an `#ifdef` around this part for conditional inclusion, to be able leave it out of the 'minimal' builds?

Reply:
Codemirror is in my opinion to big to be ever integrated in an espeasy-binary. 
So i think the links to the files should be always there regardless of the kind of build. And the acutal files can be either served  online or locally (if present).  It actually also doesn’t need a check if the files are available at all, since if not, it simply falls back to the normal textinput 

That´s what @TD-er wanted to do because i am not able to do so (or at least it will take me a long time become familiar with c++) 
And the webserver.cpp isn’t the right place anyway... i guess